src/components/Farming/DepositModal/index.js

```diff
@@ -11,13 +10,24 @@ const DepositModal = props => {
   const [approved, setApproved] = useState('')
   const [amount, setAmount] = useState('')
   const [timeLock, setTimeLock] = useState('')
-  const [timeLockMultiplier, setTimelockMultiplier] = useState()
+  const [timeLockMultiplier, setTimelockMultiplier] = useState(1)
   const imgObj = {
-    pair1: getKnownTokenImg(props.data.symbol),
-    pair2: undefined,
+    pair1:
+      props.data.pairInfo !== undefined
+        ? props.data.pairInfo.token0.logoURI
+        : undefined,
```

Comment:
There is a default image for tokens that don't have image, don't you prefeer to set this one? In the wallet page table I'm using it.